lexos/frontend/html/rolling-window.html

```diff
@@ -83,7 +82,7 @@
 
                 <label for="window-words"><input id="window-words" name="window_type" value="Words" type="radio" checked><span></span>Words</label><br>
                 <label for="characters"><input id="characters" name="window_type" value="Characters" type="radio"><span></span>Characters</label><br>
-                <label for="lines"><input id="lines" name="window_type" value="Lines" type="radio"><span></span>Lines</label>
+                <label><input id="lines" name="window_type" value="Lines" type="radio"><span></span>Lines</label>
```

Comment:
Add the `for="lines"` attribute back to the label.

Reply:
There is an issue where we cannot select the lines radio button if the `for="lines"` attribute is included (fails on both Chrome and Firefox). Likely some issue buried in our CSS. Is this something we should absolutely add for this commit?

Reply:
There was another section that used `id="lines"`, so once I changed the id name and the `label for=""` attributes, we got it working.